kolibri/plugins/user_auth/assets/src/views/SignUpPage.vue

```diff
@@ -333,6 +333,11 @@
         context:
           'Link to open the Kolibri usage and privacy modal. It will be displayed alongside the text describing collection of demographic user information.',
       },
+      /* eslint-disable kolibri/vue-no-unused-translations */
+      SignInOrCreateAccount: {
```

Comment:
Looks like this new string is not used anywhere, so we can delete (plus, we can't add any new strings now we're past string freeze!)